src/app/popular/layout.js

```diff
@@ -0,0 +1,7 @@
+import Page from "./page";
+
+export const metadata = {
+    title: "Popular Repos -RepoRater",
```

Comment:
Maybe something like this would be more descriptive?

```suggestion
    title: "Popular Repositories - RepoRater",
```
or
```suggestion
    title: "Popular Repositories on RepoRater",
```